test/otter_test.exs

```diff
@@ -87,6 +87,12 @@ defmodule OtterTest do
     7 = add_two_32!(3, 4)
   end
 
+  test "s_u8_u16_crash" do
+    t = create_s_u8_u16!()
+    {:ok, x} = receive_s_u8_u16(t)
+    IO.inspect(x)
+  end
```

Comment:
Some finding:
- this case `s_u8_u16_crash` has high probability to crash (around 25%)
- it only changes func call from `!` to ok-error from. Maybe it has something to do with the way we return the term?
- putting the case after `s_u8_u16`,  way lower probability to crash (around 5%).
- without any changes, running `mix test` will crash too.  (around 5%).

Reply:
Sorry, I cannot reproduce this in my environment...

Could you please change the CI script, and perhaps run `mix test` 10 times to reproduce this crash on CI so that we can confirm it's not because my local environment or yours is related to the issue you described?

Reply:
It reproduces in the CI

Reply:
by the way. Do you know how to get the core dump and print the stack trace in CI? I tried [this](https://gist.github.com/kudaba/53f68340bd28cd1e8824ac1e6ba8cf88) by failed

Reply:
It looks like that it always passes on macOS even we repeat it for 100 times... that's strange